### ISSUE TITLE
Milestone 1 Edits

### ISSUE THREAD

**Issue by Natalie1339** [Natalie1339/dsci522project#21]
Assumed "About" in README was the abstract/project summary. Make sure your abstract is a standalone summary description of your project! Often times this is the first thing someone who may be interested in your work will be looking at. Although I may be able to deduce what your project is about, please make sure your objectives and limitations are clearly stated!

Great work presenting clear rationale, knowledge gap that needs to be addressed, and utility of your analysis. Depending on expectation, I think a more focused rephrasing of your research question might strengthen your research question and what you are seeking to do. Rather than asking "if wine quality can be predicted using machine learning" (which is somewhat binary), consider something like "which physicochemical features are most characteristic of wine quality" or "how do specific chemical properties interact to influence quality ratings." This frames your work as generating insight about wine chemistry rather than undermining the effort and insights found in your work by simply summarizing it as just testing if ML works.

Good work on going through the workflow and describing your results in the about section and discussion in the PDF report! I would make sure to describe rationale of your model: why you selected them for your research question over other methods (-1).

Make sure to pre-determine and separate your test set from your training set before doing EDA on the training set to avoid "peeking" (-1)

Highly recommend putting figure numbers and captions for your plots. This allows easier referencing in your results (also helps me to mark the right content). If possible for polish, keep to similar figure dimensions/sizes (ie. Pearson and spearman plot is a different size from your other plots). For example, histograms can potentially be put into a 3x4 grid as one figure (privy to if histograms retain the right amount of information to display the correct intentions).
